pandas/conftest.py

```diff
@@ -338,6 +337,13 @@ def any_int_dtype(request):
     return request.param
 
 
+FLOAT_DTYPES = [float, "float32", "float64"]
+COMPLEX_DTYPES = [complex, "complex64", "complex128"]
+STRING_DTYPES = [str, 'str', 'U']
+ALL_REAL_DTYPES = FLOAT_DTYPES + ALL_INT_DTYPES
+ALL_NUMPY_DTYPES = ALL_REAL_DTYPES + STRING_DTYPES + COMPLEX_DTYPES
+
+
```

Comment:
Make sure that these lists aren't duplicated in the file.

Reply:
One block of all-caps definitions and use it everywhere in the file.  For example, search for : 

`float, "float32", "float64"`

in your version of `conftest.py` .  You'll see it exists twice in the file.

Reply:
pls put these with the other definitions